src/ILLink.RoslynAnalyzer/RequiresUnreferencedCodeAnalyzer.cs

```diff
@@ -15,8 +15,10 @@ namespace ILLink.RoslynAnalyzer
 	public class RequiresUnreferencedCodeAnalyzer : DiagnosticAnalyzer
 	{
 		public const string DiagnosticId = "IL2026";
+		const string RequiresUnreferencedCodeAttribute = nameof (RequiresUnreferencedCodeAttribute);
+		const string FullyQualifiedRequiresUnreferencedCodeAttribute = "System.Diagnostics.CodeAnalysis." + RequiresUnreferencedCodeAttribute;
 
-		private static readonly DiagnosticDescriptor s_rule = new DiagnosticDescriptor (
+		private static readonly DiagnosticDescriptor RequiresUnreferencedCodeRule = new DiagnosticDescriptor (
```

Comment:
Is there a special coding convention for private fields in the linker repo? I believe the standard `dotnet` guidelines are `s_` prefix for static privates.

Reply:
There's not AFAIK, but we can adhere to the standard dotnet guidelines.